Перший переклад

Magshifter opened the pull request:
Відправляю

Magshifter committed:
Update contribute-to-the-user-manual.md

Переклад моєї першої сторінки

Magshifter committed:
Update README.md

Ще один переклад

Magshifter committed:
Update README.md

elektro-NIK commented on edgetx-how-to/contribute-to-the-user-manual.md:
```diff
@@ -1,12 +1,12 @@
-3. To submit new How-to articles, please prepare your article in either MS word or Google Doc format. For consistency, please use screenshots from the EdgeTX simulator for pictures. Picture resolution should be 480 x 272 pixels if possible. Submit your article as an issue with the proposed article attached. From there it will be reviewed and once accepted, added to the knowledge base by one of th ...
-4. To make a recommendation for a future How-to article, please make an issue via Github.
+1. Перейдіть до нашого репозиторію на GitHub за цим посиланням: [https://github.com/EdgeTX/edgetx-user-manual](https://github.com/EdgeTX/edgetx-user-manual)
+2. Для невеликих змін у посібнику користувача або виправлень і доповнень на сторінці інструкцій, будь ласка, створіть нове звернення в GitHub і детально поясніть, що саме, на вашу думку, слід змінити та яким чином.;
+3. Для подання нових статей з інструкціями, будь ласка, підготуйте свою статтю у форматі MS Word або Google Docs. Для узгодженості використовуйте знімки екрана з симулятора EdgeTX для ілюстрацій. Роздільна здатність зображень має бути 480 x 272 пікселів, якщо це можливо. Надішліть свою статтю як звернення з прикріпленою пропозицією. Після цього її буде розглянуто, і, якщо схвалено, один із редакт ...
+4. Щоб зробити пропозицію щодо майбутньої статті з інструкціями, будь ласка, створіть звернення через GitHub.
```
Гадаю, що тут краще буде залишити термін, як в інтерфейсі GitHub за замовчуванням
```suggestion
2. Для невеликих змін у посібнику користувача або виправлень і доповнень на сторінці інструкцій, будь ласка, створіть нове Issue в GitHub і детально поясніть, що саме, на вашу думку, слід змінити та яким чином.;
3. Для подання нових статей з інструкціями, будь ласка, підготуйте свою статтю у форматі MS Word або Google Docs. Для узгодженості використовуйте знімки екрана з симулятора EdgeTX для ілюстрацій. Роздільна здатність зображень має бути 480 x 272 пікселів, якщо це можливо. Надішліть свою статтю як Issue з прикріпленою пропозицією. Після цього її буде розглянуто, і, якщо схвалено, один із редакторів додасть її до бази знань. Якщо ви бажаєте працювати над однією з запитаних інструкцій зі списку Issues, будь ласка, залиште коментар у Issue, що ви працюєте над ним, щоб ніхто інший не взявся за ту саму роботу. Після завершення прикріпіть свою роботу до Issue для перегляду.
4. Щоб зробити пропозицію щодо майбутньої статті з інструкціями, будь ласка, створіть Issue через GitHub.
```

Magshifter replied:
Так точно :)

Magshifter committed:
Update SUMMARY.md

переклав з урахуванням термінології на форумі.
Мікси - замінив на мікшування. Здалося більш логічним та інтуітивним

Magshifter committed:
Update SUMMARY.md

Допрацював

Magshifter committed:
Update edgetx-how-to/contribute-to-the-user-manual.md

Co-authored-by: elektroNIK <bogdan.kalinin@gmail.com>

elektro-NIK commented on SUMMARY.md:
```diff
@@ -1,97 +1,97 @@
+      * [Газ (Throttle)](color-radios/model-settings/model-setup/throttle.md)
+      * [Увімкнені функції](color-radios/model-settings/model-setup/enabled-features.md)
+      * [USB Джойстик](color-radios/model-settings/model-setup/usb-joystick.md)
+    * [Установка гелікоптера](color-radios/model-settings/heli-setup.md)
```
Тут певно краще налаштування
```suggestion
    * [Налаштування гелікоптера](color-radios/model-settings/heli-setup.md)
```

elektro-NIK commented on SUMMARY.md:
```diff
@@ -1,97 +1,97 @@
+      * [USB Джойстик](color-radios/model-settings/model-setup/usb-joystick.md)
+    * [Установка гелікоптера](color-radios/model-settings/heli-setup.md)
+    * [Режими польоту](color-radios/model-settings/flight-modes.md)
+    * [Входи, мікшування та виходи](color-radios/model-settings/inputs-mixes-and-outputs/README.md)
+      * [Входи](color-radios/model-settings/inputs-mixes-and-outputs/inputs.md)
+      * [Мікшування](color-radios/model-settings/inputs-mixes-and-outputs/mixes.md)
+      * [Виходи](color-radios/model-settings/inputs-mixes-and-outputs/outputs.md)
```
Тут мається на увазі назва пункту меню (іменник), а не дія
```suggestion
    * [Входи, мікшери та виходи](color-radios/model-settings/inputs-mixes-and-outputs/README.md)
      * [Входи](color-radios/model-settings/inputs-mixes-and-outputs/inputs.md)
      * [Мікшери](color-radios/model-settings/inputs-mixes-and-outputs/mixes.md)
      * [Виходи](color-radios/model-settings/inputs-mixes-and-outputs/outputs.md)
```

elektro-NIK commented on SUMMARY.md:
```diff
@@ -1,97 +1,97 @@
+      * [Звичайні телеметричні сенсори](color-radios/model-settings/telemetry/common-telemetry-sensors.md)
+  * [Налаштування апаратури](color-radios/radio-settings/README.md)
+    * [Інструменти](color-radios/radio-settings/tools.md)
+    * [SD Карата](color-radios/radio-settings/sd-card.md)
```
```suggestion
    * [SD Карта](color-radios/radio-settings/sd-card.md)
```

elektro-NIK commented on SUMMARY.md:
```diff
@@ -1,97 +1,97 @@
+  * [Налаштування апаратури](color-radios/radio-settings/README.md)
+    * [Інструменти](color-radios/radio-settings/tools.md)
+    * [SD Карата](color-radios/radio-settings/sd-card.md)
+    * [Установка апаратури](color-radios/radio-settings/radio-setup/README.md)
```
Гадаю, що тут краще буде налаштування:
```suggestion
    * [Налаштування апаратури](color-radios/radio-settings/radio-setup/README.md)
```

elektro-NIK commented on SUMMARY.md:
```diff
@@ -1,97 +1,97 @@
+* [Посібник користувача для апаратури з монохромним екраном](bw-radios/README.md)
+  * [Інтерфейс користувача](bw-radios/user-inteface.md)
+  * [Основний вигляд](bw-radios/main-view/README.md)
+    * [Перезавантаження](bw-radios/main-view/reset.md)
```
Тут мова про скидання
```suggestion
    * [Скидання](bw-radios/main-view/reset.md)
```

elektro-NIK commented on SUMMARY.md:
```diff
@@ -1,97 +1,97 @@
+    * [Статистика](bw-radios/main-view/statistics.md)
+  * [Налаштування моделі](bw-radios/model-select/README.md)
+    * [Установка](bw-radios/model-select/setup.md)
+    * [Установлення гелікоптера](bw-radios/model-select/heli-setup.md)
```
Тут певно краще налаштування
```suggestion
    * [Налаштування гелікоптера](bw-radios/model-select/heli-setup.md)
```

elektro-NIK commented on SUMMARY.md:
```diff
@@ -1,97 +1,97 @@
+    * [Установка](bw-radios/model-select/setup.md)
+    * [Установлення гелікоптера](bw-radios/model-select/heli-setup.md)
+    * [Режими польоту/їзди](bw-radios/model-select/flight-modes.md)
+    * [Входи, мікшування та виходи](bw-radios/model-select/inputs-mixes-and-outputs/README.md)
+      * [Входи](bw-radios/model-select/inputs-mixes-and-outputs/inputs.md)
+      * [Мікшування](bw-radios/model-select/inputs-mixes-and-outputs/mixes.md)
+      * [Виходи](bw-radios/model-select/inputs-mixes-and-outputs/outputs.md)
```
Тут мається на увазі назва пункту меню (іменник), а не дія
```suggestion
    * [Входи, мікшери та виходи](bw-radios/model-select/inputs-mixes-and-outputs/README.md)
      * [Входи](bw-radios/model-select/inputs-mixes-and-outputs/inputs.md)
      * [Мікшери](bw-radios/model-select/inputs-mixes-and-outputs/mixes.md)
      * [Виходи](bw-radios/model-select/inputs-mixes-and-outputs/outputs.md)
```

elektro-NIK commented on SUMMARY.md:
```diff
@@ -1,97 +1,97 @@
+    * [Екран](bw-radios/model-select/display.md)
+  * [Налаштування апаратури](bw-radios/radio-settings/README.md)
+    * [SD Карта](bw-radios/radio-settings/sd-card.md)
+    * [Установка апаратури](bw-radios/radio-settings/radio-setup.md)
```
Тут певно краще налаштування
```suggestion
    * [Налаштування апаратури](bw-radios/radio-settings/radio-setup.md)
```

Magshifter committed:
Update SUMMARY.md

Co-authored-by: elektroNIK <bogdan.kalinin@gmail.com>

Magshifter committed:
Update SUMMARY.md

Co-authored-by: elektroNIK <bogdan.kalinin@gmail.com>

Magshifter committed:
Update SUMMARY.md

Co-authored-by: elektroNIK <bogdan.kalinin@gmail.com>

Magshifter committed:
Update SUMMARY.md

Co-authored-by: elektroNIK <bogdan.kalinin@gmail.com>

Magshifter committed:
Update SUMMARY.md

Co-authored-by: elektroNIK <bogdan.kalinin@gmail.com>

Magshifter committed:
Update SUMMARY.md

Co-authored-by: elektroNIK <bogdan.kalinin@gmail.com>

Magshifter committed:
Update SUMMARY.md

Co-authored-by: elektroNIK <bogdan.kalinin@gmail.com>

Magshifter committed:
Update SUMMARY.md

Co-authored-by: elektroNIK <bogdan.kalinin@gmail.com>
